exercises/src/test/java/c7_ErrorHandling.java

```diff
@@ -126,7 +126,7 @@ public void unit_of_work() {
                 ;
 
         StepVerifier.create(taskFlux)
-                    .expectNextMatches(task -> task.executedExceptionally.get())
+                    .expectNextMatches(task -> task.executedExceptionally.get() && !task.executedSuccessfully.get())
                     .expectNextMatches(task -> task.executedSuccessfully.get())
```

Comment:
Can you add same check for other line and Il approve merge?
`.expectNextMatches(task -> task.executedExceptionally.get() && !task.executedSuccessfully.get())`
`.expectNextMatches(task -> task.executedSuccessfully.get() && task.executedSuccessfully.get())`